lib/exchange_clients/bitfinex/index.js

```diff
@@ -112,6 +121,13 @@ class BitfinexEchangeConnection {
     return socket.ws.cancelOrder(id)
   }
 
+  async getOrderHistory () {
+    const request = promisify(this.restAuth.orderHistory).bind(this.restAuth)
+
+    const res = await request(null, null, Date.now(), 10)
```

Comment:
isn't it enough to just `return request(null, null, Date(....` since you're calling it later like this `const orderHistory = await client.getOrderHistory()`

Reply:
i like this for faster debugging, its very convenient to insert a `console.log` there in this way :) Makes day to day dev a lot more efficient